engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -117,17 +115,33 @@ public SolGame(String shipName, boolean isTutorial, boolean isNewGame, CommonDra
         // TODO: make this non-static
         FactionInfo.init();
 
+        this.context = context;
+
         this.isTutorial = isTutorial;
         solApplication = context.get(SolApplication.class);
-        ModuleManager moduleManager = context.get(ModuleManager.class);
+
+        //This no longer needs to be instantiated in SolGame
```

Comment:
Should this perhaps be marked TODO? At least that's how the comment reads (Also applies to the following)